package.json

```diff
@@ -74,7 +74,9 @@
     },
     "win": {
       "target": "nsis",
-      "publisherName": "Lisk Stiftung"
+      "publisherName": "Lisk Stiftung",
+      "sign": "./scripts/sign.js",
+      "certificateSubjectName": "key_480054267"
```

Comment:
Hmm, maybe `certificateKeypairAlias`?

Reply:
No it has to be the property provided in docs https://www.electron.build/configuration/win#WindowsConfiguration-certificateSubjectName